### PR TITLE
Fixes #33181 - pagination info at the bottom & top

### PR DESCRIPTION
**Pull request by parthaa**
This commit updates the tablewrapper react component and renders the page controls on both top and bottom of the main table.

### CODE REVIEW

**Comment by parthaa** [comment]
cv-paging

File: webpack/components/Table/TableWrapper.js [hunk 1]
```diff
@@ -86,19 +102,10 @@ const TableWrapper = ({
...
-            itemCount={total}
-            page={page}
-            perPage={perPage}
-            onSetPage={(_evt, updated) => onPaginationUpdate({ page: updated })}
-            onPerPageSelect={(_evt, updated) => onPaginationUpdate({ per_page: updated })}
-            perPageOptions={usePaginationOptions().map(p => ({ title: p.toString(), value: p }))}
-            variant="top"
-          />
-        </FlexItem>
+        {ShowPageControls(PaginationVariant.top)}
```
Comment (jeremylenz) [inline webpack/components/Table/TableWrapper.js:105]: A function that returns JSX is a React component. Should be invoked not with `{...}` but instead
```js
<ShowPageControls variant={PaginationVariant.top} />
```

However, I wouldn't have this as a separate component anyway.

Comment (jeremylenz) [reply webpack/components/Table/TableWrapper.js:105]: Or you may choose to keep it as a separate component.. I don't feel as strongly about that. As long as it's notated correctly ;)

**Review by Andrewgdewar — CHANGES_REQUESTED** [review]

**Comment by jeremylenz** [comment]
Think this needs a rebase now.. sorry! :grimacing:

File: webpack/components/Table/TableWrapper.js [hunk 2]
```diff
@@ -75,6 +75,22 @@ const TableWrapper = ({
     updatePagination(updatedPagination);
   };
 
+  const ShowPageControls = variant => (
```
Comment (jeremylenz) [inline webpack/components/Table/TableWrapper.js:78]: I actually like Andrew's name better than mine:
```suggestion
  const PageControls = variant => (
```

File: webpack/components/Table/TableWrapper.js [hunk 3]
```diff
@@ -113,6 +119,7 @@ const TableWrapper = ({
       >
         {children}
       </MainTable>
+      <Flex><ShowPageControls variant={PaginationVariant.bottom} /></Flex>
```
Comment (jeremylenz) [inline webpack/components/Table/TableWrapper.js:122]: Prefer the child(ren) of `<Flex>` to be on a separate line.. just looks nicer to me

**Review by jeremylenz — COMMENTED** [review]
Looking good! Didn't test it but just leaving a couple minor code comments

**Comment by jeremylenz** [comment]
oh, looks like the tests also need to be updated!

**Comment by jeremylenz** [comment]
@parthaa I'm still seeing the full pagination component at the top. Should be the compact one at the top (without the text field)

**Review by jeremylenz — APPROVED** [review]
Nice work @parthaa 

APJ :+1:

**Comment by parthaa** [comment]
[test katello]

**Pull request merged**